tests/test_cache_lock_file_perf.py

```diff
@@ -0,0 +1,110 @@
+import multiprocessing
+import threading
+import time
+from msal_extensions import FilePersistence, CrossPlatLock
+
+
+def validate_thread_locking_file(num):
+    with open("C:/Users/abpati/Desktop/test/lockfilelog.txt") as handle:
```

Comment:
won't work on Linux :) Consider putting this in the same directory as the py script or in some other directory that is known to exist